doc/rados/operations/balancer.rst

```diff
@@ -20,70 +21,78 @@ The current status of the balancer can be checked at any time with:
 Automatic balancing
 -------------------
 
-The automatic balancing feature is enabled by default in ``upmap``
-mode. Please refer to :ref:`upmap` for more details. The balancer can be
-turned off with:
+When the balancer is in ``upmap`` mode, the automatic balancing feature is
+enabled by default. For more details, see :ref:`upmap`.  To disable the
+balancer, run the following command:
 
    .. prompt:: bash $
 
       ceph balancer off
 
-The balancer mode can be changed to ``crush-compat`` mode, which is
-backward compatible with older clients, and will make small changes to
-the data distribution over time to ensure that OSDs are equally utilized.
+The balancer mode can be changed from ``upmap`` mode to ``crush-compat`` mode.
+The ``crush-compat`` mode is backward compatible with older clients.  In
+``crush-compat`` mode, the balancer automatically makes small changes to the
+data distribution in order to ensure that OSDs are utilized equally.
 
 
 Throttling
 ----------
 
-No adjustments will be made to the PG distribution if the cluster is
-degraded (e.g., because an OSD has failed and the system has not yet
-healed itself).
+If the cluster is degraded (that is, if an OSD has failed and the system hasn't
+healed itself yet), then the balancer will not make any adjustments to the PG
+distribution.
 
-When the cluster is healthy, the balancer will throttle its changes
-such that the percentage of PGs that are misplaced (i.e., that need to
-be moved) is below a threshold of (by default) 5%.  The
-``target_max_misplaced_ratio`` threshold can be adjusted with:
+If the cluster is healthy, then the balancer will throttle its changes.  More
+specifically, the percentage of PGs that are misplaced and that need to be
```

Comment:
Seems even more stilted this way.  Suggest "When the cluster is healthy, the balancer will incrementally move a small fraction of unbalanced PGs in order to improve distribution.  This fraction will not exceed a certain threshold that deafaults to 5%."

Reply:
Yep. Your suggested text is much more natural.

Reply:
Accepted (obviously).